packs/equipment/ghost-lantern.json

```diff
@@ -34,7 +34,27 @@
             "title": "Pathfinder Treasure Vault"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "TokenLight",
+                "value": {
+                    "animation": {
+                        "intensity": 4,
+                        "speed": 1,
+                        "type": "torch"
```

Comment:
Generally avoid using the flickering light type

Reply:
I based it on bullseye lantern. Is there one you prefer?

Reply:
Updated to torch.